dashboard/src/components/typings/che.d.ts

```diff
@@ -334,7 +334,6 @@ declare namespace che {
 
   export interface IWorkspaceDevfile {
     apiVersion: string;
-    name: string;
```

Comment:
should we add here **metadata** field?

Reply:
It's there:

 https://github.com/eclipse/che/pull/13513/files/5db47044fd545e6e5b22fd248d32f3ecb305bf31#diff-eef3e1d4b54658f9d78c999b924ec5f9L342

Is it defined ok like that?

Reply:
got it, thank you. missed that , sorry